### ISSUE TITLE
Enhanced experience with chosing icons for files/folder

### ISSUE THREAD

**Issue by CuelmsI** [FlorianWoelki/obsidian-iconize#485]
**Is your feature request related to a problem? Please describe.**
When I right click on the folder for example and click change icon, there is a pop-up window with some icons. I do not see all icons as the name suggests.

**Describe the solution you'd like**
I would like to be able to scroll through all icons or click through pages, maybe managed in some way. Different tabs for each of the icon pack maybe?

**Additional context**
![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/166916034/78faaa82-f0c5-4996-bd1f-f52c976dd8b8)
![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/166916034/001eafef-821b-4e63-9920-5aa4fb3f80e4)
![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/166916034/a06cdda8-eef3-44f2-ae34-246577f6eb05)

Only Fa icons no Ra icons unless I specifcally type in search bar Ra.

![image](https://github.com/FlorianWoelki/obsidian-iconize/assets/166916034/4a00e981-cfb2-4aee-8310-8d2e87c56cf3)

**Comment by dev-bd** [comment]
Agreed, this is a make or break for me to be honest. I think there should be a setting to enable all icons while using [right click > change icon] on folders and the like. Also, unless I'm missing something you can't right click and add icons inside actual notes, so it'd be cool to enable that (even if it is essentially just a shortcut to the : identifier in the popup menu). Should be a relatively easy thing to implement, but excuse my ignorance if this isn't the case.

Looking forward to checking in on this if you're able to get around to it :)
